### DISCUSSION TITLE
AMP Validator Errors

### DISCUSSION THREAD

**Discussion by zerkshaban** [vercel/next.js#20057] [Help]
**Problem**
I am new to next/amp, I am using amp with nextjs and in dev mode amp validator is throwing errors which I am unable to wrap my head around. Following are the errors that are being thrown:

-  **CSS syntax error in tag 'style amp-custom' - saw invalid at rule '@import'** 
- **CSS syntax error in tag 'style amp-custom' - saw invalid at rule '@charset'**

**Comment by GRASPARK** [comment]
It would be good if you can share an lines of code for reference to this errors.

Basically it happened when you have call external style using **LINK tag**.

- For Example like this: `<link rel="stylesheet" href="style.min.css">`
- **Solution:-** Use INLINE Style within the page to solve this kind of errors.

If you have the same case than it would be helpful.
